src/task4/2.py

```diff
@@ -0,0 +1,29 @@
+"""Function, that receives not empty list of non-repeating integers,
+ sorted descending and "shortens" the list"""
+
+
+def get_ranges(my_list):
+    a = b = None
```

Comment:
overall it's good solution but naming like `a b i` make it completely not understandable